modules/theme-tools/featured-content.php

```diff
@@ -493,14 +493,14 @@ public static function customize_register( $wp_customize ) {
 			'priority'       => 20,
 		) );
 		$wp_customize->add_control( 'featured-content[hide-tag]', array(
-			'label'          => __( 'Hide tag from displaying in post meta and tag clouds.', 'jetpack' ),
+			'label'          => __( 'Hide tag from displaying in post details and tag clouds.', 'jetpack' ),
 			'section'        => 'featured_content',
 			'theme_supports' => 'featured-content',
 			'type'           => 'checkbox',
 			'priority'       => 30,
 		) );
 		$wp_customize->add_control( 'featured-content[show-all]', array(
-			'label'          => __( 'Display tag content in all listings.', 'jetpack' ),
+			'label'          => __( 'Display tagged posts outside the Featured Content area.', 'jetpack' ),
```

Comment:
Thanks! Changed in 9e09e7d9dcaa680cb924b374d511ba506a8ca8ae

Thanks Maui too :)

![moana-review-maui-youre-welcome-song-gif](https://cloud.githubusercontent.com/assets/426388/24045758/722b4de2-0b20-11e7-8df6-063c543a404f.gif)